docs/Framework.md

```diff
@@ -805,120 +818,100 @@ enum NcResetCause {
+    attribute NcString    contextNamespace; // Context namespace
 };
 ```
 
-### NcMethodStatus
+#### NcTouchpointNmos
 
 ```typescript
-// Method invokation status
-enum NcMethodStatus {
-    "Ok",        // 200 Method call was successful
-    "PropertyDeprecated",        // 298 Method call was successful but targeted property is deprecated
-    "MethodDeprecated",        // 299 Method call was successful but method is deprecated
-    "BadCommandFormat",        // 400 Badly-formed command
-    "Unauthorized",        // 401 Client is not authorized
-    "BadOid",        // 404 Command addresses a nonexistent object
-    "Readonly",        // 405 Attempt to change read-only state
-    "InvalidRequest",        // 406 Method call is invalid in current operating context
-    "Conflict",        // 409 There is a conflict with the current state of the device
-    "BufferOverflow",        // 413 Something was too big
-    "ParameterError",        // 417 Method parameter does not meet expectations
-    "Locked",        // 423 Addressed object is locked
-    "DeviceError",        // 500 Internal device error
-    "MethodNotImplemented",        // 501 Addressed method is not implemented by the addressed object
-    "PropertyNotImplemented",        // 502 Addressed property is not implemented by the addressed object
-    "NotReady",        // 503 The device is not ready to handle any commands
-    "Timeout",        // 504 Method call did not finish within the allotted time
-    "ProtocolVersionError"        // 505 Incompatible protocol version
+// Touchpoint class for NMOS resources
+interface NcTouchpointNmos: NcTouchpoint {
+    attribute NcTouchpointResourceNmos    resource; // Context NMOS resource
 };
 ```
 
-### NcMethodResult
+The `contextNamespace` attribute is inherited from NcTouchpoint and can only be `x-nmos`.
 
-All methods MUST return a datatype which inherits from NcMethodResult.  
-When a method call encounters an error the return MUST be [NcMethodResultError](#ncmethodresulterror) or a derived datatype.
+#### NcTouchpointNmosChannelMapping
 
 ```typescript
-// Base result of the invoked method
-interface NcMethodResult {
-    attribute NcMethodStatus    status; // Status for the invoked method
+// Touchpoint class for NMOS IS-08 resources
+interface NcTouchpointNmosChannelMapping: NcTouchpoint {
+    attribute NcTouchpointResourceNmosChannelMapping    resource; // Context Channel Mapping resource
 };
 ```
 
-### NcMethodResultError
+### NcTouchpointResource
 
 ```typescript
-// Error result - to be used when the method call encounters an error
-interface NcMethodResultError: NcMethodResult {
-    attribute NcString    errorMessage; // Error message
+// Touchpoint resource class
+interface NcTouchpointResource {
+    attribute NcString    resourceType; // The type of the resource
 };
 ```
 
-### NcMethodResultPropertyValue
+The `contextNamespace` attribute is inherited from NcTouchpoint and can only be `x-nmos/channelmapping`
```

Comment:
This needs to be moved under the NcTouchpointNmosChannelMapping section.

Reply:
Fixed in latest commit